README.md

```diff
@@ -56,6 +67,7 @@ See also [FlappyBird](https://github.com/Schmavery/FlappyBird) or [2048](https:/
 
 - There are no built-in variables like `width` and `mouseX`.  Instead, these are functions that are called, passing in an environment object that is always provided.
 ```reason
+# open Reprocessing
```

Comment:
Just remove it.

Reply:
This `#` means "have this be evaluated as part of the code, but don't display it."